backend/src/index.ts

```diff
@@ -306,6 +313,13 @@ class Server {
       this.lastHeapLogTime = now;
     }
   }
+
+  onExit(exitEvent): void {
+    DB.releasePidLock();
```

Comment:
@mononaut don't you have to check for `config.DATABASE.ENABLED` here?

Reply:
yes, I probably should.

although `DB.releasePidLock()` will have no effect unless the pid in the lock file belongs to the current process, which it shouldn't if we started with `config.DATABASE.ENABLED=false`.